src/main/java/com/haruka/helloworld/HelloWorldController.java

```diff
@@ -3,12 +3,15 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.time.LocalDateTime;
+
 @RestController
 public class HelloWorldController {
     @GetMapping("/hello")
-    public String hello() {
-        String hello = "Hello World";
-        return hello;
-    }
+    public Date date() {
 
+        Date date = new Date(LocalDateTime.now());
+        return date;

```

Comment:
現在はオリジナルのクラスを設けてインスタンスを生成してますが、
現在時刻だけなら
```suggestion
        return LocalDateTIme.now();
```
と直接返してしまってもよいかもしれませんね。